lib/generators/react_on_rails/templates/lib/tasks/assets.rake

```diff
@@ -0,0 +1,24 @@
+# lib/tasks/assets.rake
+# The webpack task must run before assets:environment task.
+# Otherwise Sprockets cannot find the files that webpack produces.
+Rake::Task["assets:precompile"]
+  .clear_prerequisites
+  .enhance(["assets:compile_environment"])
+
+namespace :assets do
+  # In this task, set prerequisites for the assets:precompile task
+  task compile_environment: :webpack do
+    Rake::Task["assets:environment"].invoke
+  end
+
+  desc "Compile assets with webpack"
+  task :webpack do
+    sh "cd client && npm run build:client"
+    sh "cd client && npm run build:server"
+  end
+
+  task :clobber do
+    rm_rf "#{RailsReactTutorial::Application.config.root}/app/assets/javascripts/generated/client-bundle.js"
```

Comment:
@justin808 @aaronvb Changes made in tutorial on https://github.com/shakacode/react-webpack-rails-tutorial/pull/129